src/libsystemd/sd-device/sd-device.c

```diff
@@ -1429,6 +1429,67 @@ _public_ int sd_device_get_seqnum(sd_device *device, uint64_t *ret) {
         return 0;
 }
 
+static int device_shall_ignore_diskseq(sd_device *device) {
+        int r;
+
+        assert(device);
+
+        /* Accessing the property will initialize the device if possible and not done. */
+        r = device_get_property_bool(device, "ID_IGNORE_DISKSEQ");
+        if (r != -ENOENT)
+                return r;
+
+        /* ENOENT means a) the device is not initialized yet and diskseq support should remain
+         * disabled b) the property is not defined,  we should try to support diskseq. */
+        return !device->is_initialized;
```

Comment:
> the device is not initialized yet and diskseq support should remain disabled

Hm? Why?

Reply:
what value of diskseq can we use if the device has not been announced by the kernel yet ?